qa/wildfly/build.gradle

```diff
@@ -55,6 +56,7 @@ war {
 elasticsearch_distributions {
   docker {
     type = 'docker'
+    architecture = Architecture.X64
```

Comment:
Same as above. Why not use current arch?

Reply:
Yes again. Both now use the current arch.